packages/react-charting/src/components/GaugeChart/GaugeChart.styles.ts

```diff
@@ -76,7 +76,7 @@ export const getStyles = (props: IGaugeChartStyleProps): IGaugeChartStyles => {
       {
         ...theme.fonts.small,
         lineHeight: '16px',
-        opacity: '0.8',
+        opacity: '0.85',
```

Comment:
test snapshots also need to be updated for this change

Reply:
updated